subprojects/packagefiles/godot-cpp/meson.build

```
@@ -1,7 +1,7 @@
 # Disclaimer: I've found this file on the internet at some point and could
 # not find it since. The original author is unknown to me :(
 
-project('godot-cpp', 'cpp', version: '4.2',
+project('godot-cpp', 'cpp', version: '4.3-0',
```

No, leave the project version unchanged:
```suggestion
project('godot-cpp', 'cpp', version: '4.3',
```

Project version != wrap release version.

---

sorry :(

Is there any way to ran ./tools/sanity_check locally only for godot ?